pykechain/models/__init__.py

```diff
@@ -38,6 +38,8 @@
     'SelectListProperty2',
     'DatetimeProperty2'
     'ActivityReferenceProperty',
+    'ScopeReferenceProperty',
```

Comment:
This is a deviation from the naming in KE-chain. We call it UserReference`s`Property and ScopeReference`s`Property. That also holds for the ActivityReference`s`property though.

I suggest to re-align this naming in pykechain. As I look at it this is just inconsistent and sloppy.

May the `MultiReferenceProperty` is out of harms way as that one is already embedded being very old.

We need to discuss with @JelleBoersma on the impact on the code of scripts.

Reply:
Discussed with @JelleBoersma , there is no impact on the ActivityReference property